static/src/js/components/PermissionForm/PermissionForm.jsx

```diff
@@ -21,22 +21,23 @@ import saveTypesToHumanizedStringMap from '@/js/constants/saveTypesToHumanizedSt
 import saveTypes from '@/js/constants/saveTypes'
 
 import { useMutation, useSuspenseQuery } from '@apollo/client'
+import { cloneDeep } from '@apollo/client/utilities'
 
 import useAppContext from '@/js/hooks/useAppContext'
 import useNotificationsContext from '@/js/hooks/useNotificationsContext'
+import useAvailableProviders from '@/js/hooks/useAvailableProviders'
 
 import errorLogger from '@/js/utils/errorLogger'
 import removeEmpty from '@/js/utils/removeEmpty'
 
 import { CREATE_ACL } from '@/js/operations/mutations/createAcl'
 import { UPDATE_ACL } from '@/js/operations/mutations/updateAcl'
-
-import CollectionSelectorPage from '@/js/pages/CollectionSelectorPage/CollectionSelectorPage'
-
-import { cloneDeep } from '@apollo/client/utilities'
 import {
   GET_COLLECTION_FOR_PERMISSION_FORM
 } from '@/js/operations/queries/getCollectionForPermissionForm'
+
```

Comment:
When we import files us `@` syntax into files that have imports not using it, we should be taking that opportunity to update the existing imports to use the newer `@` syntax.

Reply:
done